applications/nlp/transformer/evaluate_model.py

```diff
@@ -272,37 +284,38 @@ def evaluate_transformer(weights_prefix):
     """
 
     # Load model weights from file
-    embedding_layer = load_embedding_layer(weights_prefix)
-    transformer = load_transformer(weights_prefix)
-    classifier = torch.nn.Linear(embed_dim, vocab_size, bias=False)
+    embedding_layer = load_embedding_layer(weights_prefix).to(device)
+    transformer = load_transformer(weights_prefix).to(device)
+    classifier = torch.nn.Linear(embed_dim, vocab_size, bias=False).to(device)
     classifier.weight = embedding_layer.weight
 
     # Evaluate model
     bleu_scores = []
-    for batch, index_start in enumerate(range(0, num_samples, mini_batch_size)):
+    for batch, index_start in enumerate(tqdm(range(0, num_samples, mini_batch_size))):
```

Comment:
This adds `tqdm` as a requirement, which it wasn't before.

Reply:
Removed